staging-cloudfront.tf

```diff
@@ -94,4 +113,21 @@ resource "aws_cloudfront_distribution" "cid_staging" {
     response_code = 200
     response_page_path = "/index.html"
   }
-}
\ No newline at end of file
+}
+
+# Add a DNS record for the CloudFront distribution.
+resource "aws_route53_record" "imagedirectory_frontend_staging" {
+  zone_id = data.aws_route53_zone.imagedirectory_cloud.zone_id
+  name    = local.imagedirectory_domain_staging
```

Comment:
I _think_ this may need to just be `staging` here, but terraform/route53 may be smart enough to figure this out on its own. :wink: